packages/plugins/urlWriteback/mod.ts

```diff
@@ -25,16 +25,27 @@ interface WritebackPayload {
  * as a handler for the publishing system onUrlChange event in order to
  * save published urls back into the Knowledge Graph.
  *
+ * The fieldToFeatureMap is an optional parameter that can be used to define
+ * a custom field to store other features that are powered by entity data
+ * i.e., custom review forms that are powered by entity data but have a different
+ * CF to store the prod URL of the page.
+ * 
  * @public
  */
-export default function urlWriteback(config: UrlWritebackConfig) {
+export default function urlWriteback(config: UrlWritebackConfig, fieldToFeatureMap: Record<string, string> = {}) {
   function onUrlChange(event: WritebackPayload) {
     if (!(event.entityId && event.locale && event.url)) {
       return null;
     }
 
+    let field = config.field;
+
+    if (fieldToFeatureMap && fieldToFeatureMap[event.feature]) {
+      field = fieldToFeatureMap[event.feature];
```

Comment:
This might just be me but when I read `fieldToFeatureMap` I assumed the keys would be fields and the values would be features.

Reply:
yeah i realized that too, I'm updating the name to be `featureToFieldMap`